src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview.tsx

```diff
@@ -0,0 +1,263 @@
+import {
+  Avatar,
+  Text,
+  Box,
+  VStack,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TableContainer,
+  HStack,
+  Icon,
+  Spacer,
+  Flex,
+  Link,
+  Breadcrumb,
+  BreadcrumbItem,
+  useTheme,
+} from "@chakra-ui/react"
+import {
+  Searchbar,
+  IconButton,
+  useSearchbar,
+} from "@opengovsg/design-system-react"
+import {
+  BiGitCompare,
+  BiWorld,
+  BiFileBlank,
+  BiImage,
+  BiCog,
+  BiShow,
+  BiSort,
+  BiFilterAlt,
+  BiChevronRight,
+} from "react-icons/bi"
+
+import { BxFileArchiveSolid } from "assets"
+
+interface DisplayedDateTime {
+  date: string
+  time: string
+}
+
+const getDateTimeFromUnixTime = (unixTime: number): DisplayedDateTime => {
+  const date = new Date(unixTime)
+  return {
+    date: date.toLocaleDateString("en-GB", {
+      month: "short",
+      year: "numeric",
+      day: "numeric",
+    }),
+    time: date.toLocaleTimeString("en-GB", {
+      timeStyle: "short",
+    }),
+  }
+}
+
+const getIcon = (iconTypes: EditedItemProps["type"]): JSX.Element => {
+  const iconType = iconTypes[0]
+  switch (iconType) {
+    case "nav": {
+      return <Icon as={BiWorld} />
+    }
+    case "file": {
+      return <Icon as={BxFileArchiveSolid} />
+    }
+    case "setting": {
+      return <Icon as={BiCog} />
+    }
+    case "page": {
+      return (
+        <Icon
+          ml="0.75rem"
+          fontSize="1.25rem"
+          fill="icon.alt"
+          as={BiFileBlank}
+        />
+      )
+    }
+    case "image": {
+      return <Icon as={BiImage} />
+    }
+    default: {
+      // NOTE: This is done to ensure exhaustive type matching
+      const error: never = iconType
+      throw new Error(`Unmatched fileType: ${error}`)
```

Comment:
this isn't casting! what i'm doing here is that i'm saying that the **type of `error` is `never`**. this is similar to eg: 
```c
int x = 1;  
```

in c, where the `int` specifies the type of the variable x. whether `iconType` will be `null` is dependent upon the runtime - at compile time, ts' type system verifies the correctness of this type. at runtime, if there is `js` interop, then it really depends but because the API BE isn't specified yet so as of now, no. in the future, the API fetch will be in `ts` so barring implicit casts/conversions, there shouldn't be